### PR TITLE
Update formatting for xmls and gradle

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: buildSrc/src/main/resources/resource_file_template.txt [hunk 1]
```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  Backpack for Android - Skyscanner's Design System
+    Backpack for Android - Skyscanner's Design System
```
Comment (bvitaliyg) [inline buildSrc/src/main/resources/resource_file_template.txt:3]: Important

File: buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDimensions.kt [hunk 2]
```diff
@@ -111,5 +111,5 @@ private fun toCompose(
 
 private fun toXml(source: BpkDimensions, type: String): String =
     source.map { (name, value) ->
-        "  <dimen name=\"$type${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}\">${value}dp</dimen>"
+        "    <dimen name=\"$type${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}\">${value}dp</dimen>"
```
Comment (bvitaliyg) [inline buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDimensions.kt:114]: Important

File: buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDurations.kt [hunk 3]
```diff
@@ -66,5 +66,5 @@ private fun parseIntegers(
 
 private fun toXml(source: BpkDurations, type: String): String =
     source.map { (name, value) ->
-        "  <integer name=\"$type${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}\">$value</integer>"
+        "    <integer name=\"$type${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}\">$value</integer>"
```
Comment (bvitaliyg) [inline buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkDurations.kt:69]: Important

File: buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkIcons.kt [hunk 4]
```diff
@@ -189,7 +189,10 @@ private fun toXml(source: BpkIcons, rootDir: String, metadataPath: String): Map<
 
     fun BpkIcon.fileContent(metadata: Map<String, String?>) =
         value
-            .replace("android:fillColor=\"#FF000000\"", "android:fillColor=\"@color/bpkTextPrimary\"")
+            .replace("android:fillColor=\"#FF000000\"", "  android:fillColor=\"@color/bpkTextPrimary\"")
```
Comment (bvitaliyg) [inline buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkIcons.kt:192]: Important

File: buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextUnits.kt [hunk 5]
```diff
@@ -131,5 +131,5 @@ private fun toXml(
     source: BpkTextUnits,
 ): String =
     source.map { (name, value) ->
-        "  <dimen name=\"bpkText${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}Size\">${value.toInt()}sp</dimen>"
+        "    <dimen name=\"bpkText${CaseFormat.UPPER_UNDERSCORE.to(CaseFormat.UPPER_CAMEL, name)}Size\">${value.toInt()}sp</dimen>"
```
Comment (bvitaliyg) [inline buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkTextUnits.kt:134]: Important

**Review by marianeum — APPROVED** [review]

**Pull request merged**